asposeimagingcloud/api/imaging_api.py

```diff
@@ -0,0 +1,1459 @@
+# coding: utf-8
+# -----------------------------------------------------------------------------
+# <copyright company="Aspose" file="ImagingApi.py">
+#   Copyright (c) 2019 Aspose Pty Ltd. All rights reserved.
+# </copyright>
+# <summary>
+#   Permission is hereby granted, free of charge, to any person obtaining a
+#  copy  of this software and associated documentation files (the "Software"),
+#  to deal  in the Software without restriction, including without limitation
+#  the rights  to use, copy, modify, merge, publish, distribute, sublicense,
+#  and/or sell  copies of the Software, and to permit persons to whom the
+#  Software is  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all  copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#  FROM,  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#  DEALINGS IN THE SOFTWARE.
+# </summary>
+# -----------------------------------------------------------------------------
+from __future__ import absolute_import
+
+import six
+from asposeimagingcloud.rest import ApiException
+from asposeimagingcloud.api_client import ApiClient
+
+
+class ImagingApi(object):
+    """
+    Aspose.Images for Cloud API
```

Comment:
@s-zubov Please, change the product name to Aspose.Imaging Cloud as it's the actual one, and check for other possible mentionings of product name.